gpflow/kernels/stationaries.py

```diff
@@ -115,6 +115,12 @@ class AnisotropicStationary(Stationary):
     input dimension.
     """
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if tf.size(self.lengthscales) > 1:
```

Comment:
I would prefer to do it like we do it in `ard`:
```suggestion
  if self.lengthscales.shape.ndims > 1
```

Reply:
Nice, I've updated to just use self.ard itself